relay-kafka/src/producer/utils.rs

```diff
@@ -20,7 +20,7 @@ impl ClientContext for Context {
         relay_statsd::metric!(gauge(KafkaGauges::MessageSize) = statistics.msg_size);
         relay_statsd::metric!(gauge(KafkaGauges::MessageSizeMax) = statistics.msg_size_max);
 
-        for broker in statistics.brokers.values() {
+        for (_, broker) in statistics.brokers {
```

Comment:
Why this change?

Reply:
It was either this or `if let Some(int_latency) = &broker.int_latency {`, and I thought this function might as well consume the object.